inlong-sort/sort-connectors/postgres-cdc/src/main/java/org.apache.inlong.sort.cdc.postgres/DebeziumSourceFunction.java

```diff
@@ -423,11 +424,13 @@ public void run(SourceContext<T> sourceContext) throws Exception {
             String groupId = inlongMetricArray[0];
             String streamId = inlongMetricArray[1];
             String nodeId = inlongMetricArray[2];
-            metricData = new MetricData(metricGroup);
-            metricData.registerMetricsForNumRecordsIn(groupId, streamId, nodeId, "numRecordsIn");
-            metricData.registerMetricsForNumBytesIn(groupId, streamId, nodeId, "numBytesIn");
-            metricData.registerMetricsForNumBytesInPerSecond(groupId, streamId, nodeId, "numBytesInPerSecond");
-            metricData.registerMetricsForNumRecordsInPerSecond(groupId, streamId, nodeId, "numRecordsInPerSecond");
+            sourceMetricData = new SourceMetricData(metricGroup);
+            sourceMetricData.registerMetricsForNumRecordsIn(groupId, streamId, nodeId, "numRecordsIn");
+            sourceMetricData.registerMetricsForNumBytesIn(groupId, streamId, nodeId, "numBytesIn");
```

Comment:
"numBytesIn" can be extracted to constant

Reply:
DONE